devel/standard/standard.yaml

```diff
@@ -2,13 +2,13 @@ start:
   args:
   - firehose
   - merger
-  - extractor-node
+  - reader-node
   - relayer
   flags:
     # Specifies the path to the binary
-    extractor-node-path: "dummy-blockchain"
+    reader-node-path: "dummy-blockchain"
     # Flags that will be added to the dummy chain process command
-    extractor-node-arguments: +--dm-enabled --block-rate=6
+    reader-node-arguments: +--dm-enabled --block-rate=6
```

Comment:
`dm-enabled` flag in `dummy-blockchain` will need to change to `firehose-enabled` also (use the flag as defined in the conventions doc).